src/www/ui/api/Models/Upload.php

```diff
@@ -107,6 +113,7 @@ public function getArray()
       "description" => $this->description,
       "uploadname"  => $this->uploadName,
       "uploaddate"  => $this->uploadDate,
+      "assigneeid"  => $this->assigneeId,
```

Comment:
IMO, calling it `assignee` will be much better.

Reply:
Edited